boards/arduino/portenta_c33/arduino_portenta_c33_defconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Arduino SA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=200000000
+
+CONFIG_GPIO=y
+
+CONFIG_BUILD_OUTPUT_BIN=y
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_CLOCK_CONTROL=y
+
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+
+CONFIG_USE_DT_CODE_PARTITION=y
```

Comment:
```suggestion
```

Reply:
This needs to be set to preserve the default functionality (with dfu bootloader). Is there another way to achieve the same result?